consensus/istanbul/backend/api.go

```diff
@@ -197,12 +208,34 @@ func (api *APIExtension) GetCommittee(number *rpc.BlockNumber) ([]common.Address
 		return nil, errNoBlockExist
 	}
 
-	istanbulExtra, err := types.ExtractIstanbulExtra(header)
-	if err == nil {
-		return istanbulExtra.Validators, nil
-	} else {
-		return nil, errExtractIstanbulExtra
+	blockNumber := header.Number.Uint64()
+	round := header.Round()
+	view := &istanbul.View{
+		Sequence: new(big.Int).SetUint64(blockNumber),
+		Round:    new(big.Int).SetUint64(uint64(round)),
+	}
+
+	// get the proposer of this block.
+	proposer, err := ecrecover(header)
+	if err != nil {
+		return nil, err
+	}
+
+	// get the snapshot of the previous block.
+	parentHash := header.ParentHash
+	snap, err := api.istanbul.snapshot(api.chain, blockNumber-1, parentHash, nil)
```

Comment:
Why did you load a snapshot of the previous block instead of the target block?

Reply:
To get the committee list to validate the target block, the parent snapshot is needed.
(I noticed it by seeing the verifySigner/VerifyCommittedSeals/CheckInSubList)

Reply:
Oh.. I see. The sublist is selected based on the previous block's validator set. The method name confuses me. I think `SubListWithProposer` should be changed to `NextBlockCommittee` or something else.